src/components/TextComponent/Text.tsx

```diff
@@ -0,0 +1,21 @@
+import classNames from "classnames"
+import { ReactNode } from "react"
+
+import styles from "./text.module.scss"
+
+interface IPropsText {
+  bold?: boolean
+  children: ReactNode
+  size: "S" | "M" | "L"
+}
+const Text = ({ children, size, bold }: IPropsText): JSX.Element => {
+  const classes = classNames(
+    { [styles.small]: size === "S" },
+    { [styles.medium]: size === "M" },
+    { [styles.large]: size === "L" },
+    { [styles.bold]: bold === true },
```

Comment:
Не надо явно проверять на true. 

```suggestion
    { [styles.bold]: bold },
```